Makefile

```diff
@@ -3,10 +3,24 @@
 # remember the right command. So let's keep this simple, just targets and
 # commands.
 
+# Ensure bash shell! Needed for checking go version...
+SHELL := /bin/bash
+
 # Ensure go-extpoints and go-import-subtree are available for go generate
 export PATH := $(GOPATH)/bin:$(PATH)
 
-all: build
+# For checking go compiler has a suitable version number
+GO_VERSION := $(shell go version 2>/dev/null | cut -f3 -d' ')
```

Comment:
I would go with `awk` instead of `cut` here because of its relaxed spacing matching pattern.